plugins/inputs/bond/bond_notlinux.go

```diff
@@ -0,0 +1,4 @@
+//go:build !linux
+// +build !linux
+
+package bond
```

Comment:
Maybe we leave this as is, as there might be other OSes that also support this. Sorry for the noise!